src/Picasso_UniformMesh.hpp

```diff
@@ -81,34 +81,28 @@ class UniformMesh
         _minimum_halo_width = minimum_halo_cell_width;
 
         // Get the mesh parameters.
-        const auto& mesh_params = ptree.get_child( "mesh" );
+        auto mesh_params = inputs["mesh"];
```

Comment:
You make `const auto&` in some places and just `auto` in others.

Reply:
Will fix